docs/UserGuide.md

```diff
@@ -685,9 +685,34 @@ If your changes to the data file makes its format invalid, TaskHub will discard
 # FAQ
 
 **Q**: Why is all my TaskHub data gone?<br>
-**A**: TaskHub requires all data stored in the data file to be in the correct format so that it can be read back to TaskHub. A TaskHub that suddenly becomes empty indicates an issue with the data file.<br>
+**A**: Most likely, your data is still mostly intact, so it is recommended to exit the app first as executing some commands such as `addE` will clear the previously stored data in this situation.
+Taskhub may appear empty as it requires all data stored in the data file to be in the correct format so that it can be read back to TaskHub. A TaskHub that suddenly becomes empty indicates an issue with the data file.<br>
+
+<div markdown="span" class="alert alert-info">
+
+:information_source:
+** The solution below assumes that you are familiar with files in the `.json` format, and how TaskHub stores data. If you are unfamiliar, please check out **[this website](https://www.w3schools.com/js/js_json_intro.asp)** to know more about `.json` files ** <br>
 If you have directly changed any data in the `taskhub.json` file instead of through TaskHub, it is highly likely that it would be in the wrong format, resulting in an empty TaskHub.<br>
-If you have not changed anything, ensure your `data` folder containing your `taskhub.json` is in the same folder as your `taskhub.jar`.
+To attempt to recover as much data as possible, it is recommended to save a copy of the `taskhub.json` file somewhere else and perform the following actions:<br>
+* Remove all projects from your `taskhub.json` file. (`"projects" : []` should be the result after this operation) 
+* Run TaskHub with the edited `taskhub.json` file. If the list of employees is shown, then the problem lies in your projects. Otherwise, a format error lies in the `"employee"` section of your json file. 
+It is recommended to start from scratch because most likely your projects and tasks that involve the employee have different degree of changes, and it might be impractical to manually change each occurence of the employee.  
+* Now, add each project one by one, ensuring that TaskHub can run normally after each project addition. If TaskHub appears empty after the addition of a particular project, Congratulations! 
```

Comment:
`Congratulations!` may seem a bit misleading, perhaps you could say something like:

`then the problem lies in that project`